[2024-08-07T10:48:56.351+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T10:48:56.371+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_2024-08-07.validate_the_api scheduled__2024-05-24T00:00:00+00:00 [queued]>
[2024-08-07T10:48:56.375+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_2024-08-07.validate_the_api scheduled__2024-05-24T00:00:00+00:00 [queued]>
[2024-08-07T10:48:56.375+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-07T10:48:56.396+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): validate_the_api> on 2024-05-24 00:00:00+00:00
[2024-08-07T10:48:56.400+0000] {standard_task_runner.py:63} INFO - Started process 2215 to run task
[2024-08-07T10:48:56.402+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_2024-08-07', 'validate_the_api', 'scheduled__2024-05-24T00:00:00+00:00', '--job-id', '222', '--raw', '--subdir', 'DAGS_FOLDER/final_etl.py', '--cfg-path', '/tmp/tmpau4ohrps']
[2024-08-07T10:48:56.403+0000] {standard_task_runner.py:91} INFO - Job 222: Subtask validate_the_api
[2024-08-07T10:48:56.470+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_2024-08-07.validate_the_api scheduled__2024-05-24T00:00:00+00:00 [running]> on host 8cffaad11d99
[2024-08-07T10:48:57.021+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_2024-08-07' AIRFLOW_CTX_TASK_ID='validate_the_api' AIRFLOW_CTX_EXECUTION_DATE='2024-05-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-24T00:00:00+00:00'
[2024-08-07T10:48:57.022+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T10:48:57.033+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-07T10:48:57.045+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/data/api_utils.py']
[2024-08-07T10:48:57.051+0000] {subprocess.py:86} INFO - Output:
[2024-08-07T10:48:57.330+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-07T10:48:57.331+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T10:48:57.697+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_2024-08-07, task_id=validate_the_api, run_id=scheduled__2024-05-24T00:00:00+00:00, execution_date=20240524T000000, start_date=20240807T104856, end_date=20240807T104857
[2024-08-07T10:48:58.180+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-07T10:48:58.204+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-07T10:48:58.252+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-08-07T10:55:08.663+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-08-07T10:55:08.682+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_2024-08-07.validate_the_api scheduled__2024-05-24T00:00:00+00:00 [queued]>
[2024-08-07T10:55:08.686+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_2024-08-07.validate_the_api scheduled__2024-05-24T00:00:00+00:00 [queued]>
[2024-08-07T10:55:08.686+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-08-07T10:55:08.716+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): validate_the_api> on 2024-05-24 00:00:00+00:00
[2024-08-07T10:55:08.719+0000] {standard_task_runner.py:63} INFO - Started process 3012 to run task
[2024-08-07T10:55:08.727+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'etl_2024-08-07', 'validate_the_api', 'scheduled__2024-05-24T00:00:00+00:00', '--job-id', '440', '--raw', '--subdir', 'DAGS_FOLDER/final_etl.py', '--cfg-path', '/tmp/tmpmuwt82_6']
[2024-08-07T10:55:08.729+0000] {standard_task_runner.py:91} INFO - Job 440: Subtask validate_the_api
[2024-08-07T10:55:08.777+0000] {task_command.py:426} INFO - Running <TaskInstance: etl_2024-08-07.validate_the_api scheduled__2024-05-24T00:00:00+00:00 [running]> on host 8cffaad11d99
[2024-08-07T10:55:08.872+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='etl_2024-08-07' AIRFLOW_CTX_TASK_ID='validate_the_api' AIRFLOW_CTX_EXECUTION_DATE='2024-05-24T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-05-24T00:00:00+00:00'
[2024-08-07T10:55:08.873+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-08-07T10:55:08.880+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-08-07T10:55:08.881+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python3 /opt/***/data/api_utils.py']
[2024-08-07T10:55:08.888+0000] {subprocess.py:86} INFO - Output:
[2024-08-07T10:55:09.141+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-08-07T10:55:09.142+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-08-07T10:55:09.184+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=etl_2024-08-07, task_id=validate_the_api, run_id=scheduled__2024-05-24T00:00:00+00:00, execution_date=20240524T000000, start_date=20240807T105508, end_date=20240807T105509
[2024-08-07T10:55:09.297+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-08-07T10:55:09.323+0000] {taskinstance.py:3503} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-08-07T10:55:09.345+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
